core/test/element_unary_operations_test.cpp

```diff
@@ -280,3 +280,51 @@ TYPED_TEST(ElementNegativeInfToNumTest, value_and_variance_out) {
   targetted_replacement_out_arg_test(element::negative_inf_to_num_out_arg, out,
                                      replaceable, nonreplaceable, replacement);
 }
+
+TEST(ElementReciprocalTest, unit) {
+  const units::Unit one_over_m(units::dimensionless / units::m);
+  EXPECT_EQ(element::reciprocal(one_over_m), units::m);
```

Comment:
Could add a second test here, e.g., with `units::s` to ensure that `element::reciprocal` does not simply always return `units::m`.